src/transactions/transactionBytes.js

```diff
@@ -332,6 +321,7 @@ function createTransactionBuffer(transaction, options) {
 /**
  * @method getBytes
  * @param transaction Object
+ * @param options
```

Comment:
Is this parameter needed here now that it's been removed from `createTransactionBuffer`?